as_ikea.py

```diff
@@ -46,9 +47,8 @@ def as_ikea(name: str) -> str:
     Convert *name* to Ikea-like name.
     """
     # Replace just once
-    name=name.lower()
     index = random.choice(positions(name))
-    name = replace_at(name, index)
+    name = replace_at(name.lower(), index)
```

Comment:
We need .lower() for positions() as well, so initial option with `name=name.lower()` is correct.